docs/guide/personalization/enabling_personalization.md

```diff
@@ -0,0 +1,54 @@
+# Enabling Personalization
+
+The Ibexa Personalization solution is based on a client-server architecture.
+To make it operational, you must obtain and set up certain authentication parameters.
+
+## Requesting access to the server
+
+At first you must request credentials required to access the recommendation server.
+
+To do this, in the Back Office, select **Personalization** and then **Dashboard**.
+On the welcome screen, enter the following details in their respective fields:
+
+* a full name of the person responsible for accepting the terms and conditions of the Personalization service
+* an e-mail address to which you want the credentials to be sent to
+* an Installation key that can be found on the **Maintenance and Support agreement details** page in the service portal
```

Comment:
```suggestion
- installation key that can be found on the **Maintenance and Support agreement details** page in the service portal
```

Reply:
A capitalized version of "Installation key" happened elsewhere in the docs, hence here.
https://www.google.com/search?q=site:https://doc.ibexa.co+%22installation+key%22

Reply:
Hm, ideally both should be lowercase, but let's keep it as is for now for consistency.